src/common/appInfo.ts

```diff
@@ -0,0 +1,19 @@
+// tslint:disable-next-line:no-var-requires
+const packageJson = require("../../package.json");
```

Comment:
Instead of explicitly importing `package.json` in multiple places, refactored this into a const that can be easily imported by other modules.